.github/workflows/build-and-push-component.yaml

```diff
@@ -24,34 +20,24 @@ on:
         required: true
         type: string
         description: 'The build timestamp to be used for binaries.'
+      bake_target:
+        required: true
+        type: string
+        description: 'Docker Bake target name.'
+      cache-name-prefix:
+        required: true
+        type: string
+        description: 'A prefix for image names and digests to be cached temporarily.'
```

Comment:
Any particular reason why it's required to set the cache name? Can this not be done automatically?

Reply:
CI changes will be part of another PR

Reply:
could you please collect these comments that are planned for another PR in a separate issue @akijakya